.env.example

```diff
@@ -0,0 +1,12 @@
+#preview
+#generate keys by following README
+PUBLIC_KEY='key'
+SECRET_KEY='key'
+
+CARDANO_NODE_SOCKET_PATH="/Users/{user}/.cardano-up/state/0/preview/node.socket"
```

Comment:
This environment variable does not directly apply to this project. Maybe adding this to `zhrrc` or `bashrc` is better?

Reply:
I don't think you have to add it there, since if you create .env with these values it will be loaded when you start the application.